<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigInfo</class>
 <widget class="QWidget" name="ConfigInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>190</width>
     <height>65</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelDefaultCityTitle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>设置默认城市</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDefaultCity">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>28</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color: rgb(44, 44, 44);
	background-color: rgb(234,237,238);
	border-radius:5px;
	border:2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string>桂林市</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>默认城市</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetDefaultCity">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>29</y>
      <width>30</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    color:rgb(44,44,44);
	background-color: rgb(234,237,238);
	border:0px;
	border-radius:5px;
}

QPushButton:pressed{
 	color:rgb(234,237,238);
	background-color:rgb(44,44,44);
}	</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/res/锁定-黑色.png</normaloff>:/images/res/锁定-黑色.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>29</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/images/res/坐标-白色.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>190</width>
     <height>105</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="cBoxAutoUpdateWeather">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>默认5min/次</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>自动更新天气(勿关)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBoxCloseExitApp">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>缩小窗口退出应用</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBoxSmallWindow">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>右击菜单转换</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>开启小窗口模式</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cBoxAutoSaveMemo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>55</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>默认15min/次</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>自动保存memo文件</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>75</y>
     <width>195</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelUpdateCycleTitle">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>10</y>
      <width>38</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;color: rgb(255, 255, 255);&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>模式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comBoxModel">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>10</y>
      <width>145</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>启动默认显示</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    border: 2px solid gray;   /* 边框 */
    border-radius: 4px;   /* 圆角 */
    padding: 1px 18px 2px 3px;   /* 字体填衬 */
    color: rgb(0,0,0);
	background-color:rgb(234,237,238); 
}

/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 2px solid gray;   /* 选定项的虚框 */
    color: black;
    border: 2px solid gray;   /* 整个下拉窗体的边框 */
    background-color:rgb(234,237,238);    /* 整个下拉窗体的背景色 */
	selection-background-color: rgb(44,44,44);
}

/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover { 
	color: white;
    background-color:rgb(44,44,44);   /* 整个下拉窗体越过每项的背景色 */
}

/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
	color:white;
	background-color:rgb(44,44,44); 
}

</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLength</enum>
    </property>
    <item>
     <property name="text">
      <string>标准模式</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/res/Application.ico</normaloff>:/images/res/Application.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天气小窗口</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/res/Application.ico</normaloff>:/images/res/Application.ico</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>备忘录窗口</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/res/memo.png</normaloff>:/images/res/memo.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSure">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>140</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color:rgb(44,44,44);
	background-color: rgb(234,237,238);
	border:0px;
	border-radius:5px;
}

QPushButton:pressed{
 	color:rgb(234,237,238);
	background-color:rgb(44,44,44);
}	</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>140</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color:rgb(44,44,44);
	background-color: rgb(234,237,238);
	border:0px;
	border-radius:5px;
}

QPushButton:pressed{
 	color:rgb(234,237,238);
	background-color:rgb(44,44,44);
}	</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>5</y>
     <width>195</width>
     <height>65</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelMemoTitle_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>文件保存模式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnFileCover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>覆盖</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnFileAppend">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>追加</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtnJunk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234,237,238);</string>
    </property>
    <property name="text">
     <string>废弃</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConfigInfo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
